src/components/main/note/edit-view/EditViewTextFields.js

```diff
@@ -8,19 +8,35 @@ const useStyles = makeStyles(() => ({
   container: { display: 'flex', flexDirection: 'column' },
 }));
 
-const EditViewTextFields = ({ height }) => {
+const EditViewTextFields = ({ height, title, description, onChange }) => {
   const classes = useStyles();
 
+  let editedTitle = title;
+  let editedDescription = description;
```

Comment:
Probably you don't need these variables.

Reply:
Certainly not.